bin/restore-db

```diff
@@ -1,28 +0,0 @@
-#!/bin/bash
```

Comment:
What does this file do/ why do we no longer need it?

Reply:
It used to be able to restore the database of a PaaS environment, so it'll no longer work on AKS. We should have an alternative, but that's for a separate PR.